examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -53,70 +72,84 @@ my_module/__init__.py        1      0   100%
-A cloud builder is just a docker container with the stuff you need in it. Google doesn't distribute a 
-python cloud builder, so we have to build one ourselves. I've created one in '/python-cloud-builder.' All you need to do 
-is run this:
+Run these commands from the root of the project to build and upload your custom Python cloud build container:
 
 ```
 cd python-cloud-builder
 gcloud builds submit --config=cloudbuild.yaml .
-
 ```
 
-This creates a custom cloud builder in your container registry called gcr.io/$PROJECT_ID/python-cloudbuild that you can
-use to run tests with.
+This creates the custom Python cloud builder and uploads it to your GCP project's container registry, which is a private
+location to store container images. Your new cloud builder will be called gcr.io/$PROJECT_ID/python-cloudbuild, where 
+$PROJECT_ID is the name of your GCP project.
 
 ## 4. Create a Cloud Build Trigger
-* On the GCP console navigate to 'Cloud Build' > 'Triggers'
-* Add a trigger
-* Choose cloud source repository
+Now that you've created a Python cloud builder to run your tests, you 
+should [create a trigger](https://cloud.google.com/cloud-build/docs/running-builds/automate-builds) that tells Cloud
+Build when to run those tests. To do that, follow these steps:
+
+* On the GCP console navigate to 'Cloud Build' > 'Triggers.'
+* Add a trigger.
+* Choose cloud source repository.
 * Pick the repo you created in step 1.
-* Assuming you want the trigger to fire on any branch, accept the default trigger type and regex
-* Choose 'cloud build configuration file' from 'build configuration'
-* Click create trigger
+* Assuming you want the trigger to fire on any branch, accept the default trigger type and regex.
+* Choose 'cloud build configuration file' from 'build configuration.'
+* Click create trigger.
 
 
 ## 5. Create a cloudbuild.yaml file that executes your tests and runs pylint
+At this point you've created a Python cloud build container in step 3, a build trigger that runs whenever you 
+push new code in step 4. In this last step you'll create a set of instructions called a...
 Create a file called cloudbuild.yaml in the root of your project and add the steps needed to execute your unit tests, 
 using the cloud builder we created in step 3. 
 
-*Note: Each cloudbuilder step is a separate, ephemerial run of a docker container, however the /workspace/ directory is
-preserved between runs. One way to carry python packages forward is to use a virtualenv housed in /workspace/*
+*Note: Each cloudbuilder step is a separate, ephemerial run of a docker container, however [the /workspace/ directory is
+preserved between runs](https://cloud.google.com/cloud-build/docs/build-config#dir). One way to carry python packages 
+forward is to use a virtualenv housed in /workspace/*
 
 
 ```
 steps:
-- name: 'gcr.io/$PROJECT_ID/python-cloudbuild'
+- name: 'gcr.io/$PROJECT_ID/python-cloudbuild' # Cloud Build automatically substitutes $PROJECT_ID for your Project ID
```

Comment:
Comments should end with a period, this is in some of our style guides. Applies to the comments below as well. Some languages also specify two spaces after the end of the code before the comment, rather than one space. I'm not sure where yaml falls since we don't have a yaml guide, so I'll leave that change up to you.

Reply:
ok, I think I got these covered.